installer/scripts/omsadmin.sh

```diff
@@ -29,6 +29,7 @@ INSTALL_INFO=/etc/opt/microsoft/omsagent/sysconf/installinfo.txt
 # Ruby helpers
 RUBY=/opt/microsoft/omsagent/ruby/bin/ruby
 AUTH_KEY_SCRIPT=/opt/microsoft/omsagent/bin/auth_key.rb
+TELEMETRY_SCRIPT=/opt/microsoft/omsagent/bin/telemetry.rb
```

Comment:
We won't be able to update the files in the folder from DSC. I think we need to move it to the plugin folder. What's your opinion?